server/src/main/java/org/apache/gravitino/server/web/rest/SchemaOperations.java

```diff
@@ -184,10 +217,15 @@ public Response alterSchema(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "drop-schema." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "drop-schema", absolute = true)
+  @AuthorizationExpression(
+      expression = "METALAKE::OWNER || CATALOG::OWNER || SCHEMA::OWNER",
```

Comment:
We should have the privilege `use catalog` first.

Reply:
fixed